src/commands/subcommands/misc/handleEcho.ts

```diff
@@ -0,0 +1,28 @@
+import { CommandHandler } from "../../../interfaces/commands/CommandHandler";
+import { errorEmbedGenerator } from "../../../modules/commands/errorEmbedGenerator";
+import { beccaErrorHandler } from "../../../utils/beccaErrorHandler";
+
+/**
+ * Echos what the user type.
+ */
+export const handleEcho: CommandHandler = async (Becca, interaction, t) => {
+  try {
+    interaction.ephemeral = true;
```

Comment:
```suggestion
    await interaction.deferReply({ ephemeral: true });
```

This should be a deferred reply.